.chloggen/component-nodes.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. otlpreceiver)
+component: service
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Extract service.pipelines interface, add skeleton graph implementation
+
+# One or more tracking issues or pull requests related to the change
+issues: [6764]
+
+# (Optional) One or more lines of additional information to render under the primary note.
+# These lines will be padded with 2 spaces and then inserted directly into the document.
+# Use pipe (|) for multiline entries.
+subtext:
```

Comment:
I don't think we need changelog entries for this internal things that do not touch the logic. It will make the CHANGELOG very hard to digest since this is not important for anyone using the core collector artifacts.

Reply:
cc @codeboten @dmitryax

Reply:
Removed